ci/Dockerfile.gitlab

```diff
@@ -91,7 +91,7 @@ ENV PATH $PATH:$ANDROID_HOME/platform-tools
 ENV PATH $PATH:$ANDROID_SDK_ROOT/build-tools/${ANDROID_BUILD_TOOLS}:$ANDROID_NDK
 
 # Install Node
-ENV NODENV_VERSION 16.13.0
+ENV NODENV_VERSION 22.19.0
```

Comment:
you also need to bump Docker image version used here https://github.com/DataDog/dd-sdk-android/blob/8a51cb566a372e2c28738a014c554ffbc9644b6b/.gitlab-ci.yml#L2 and run a manual job to generate it